recipes/c-client/all/test_package/test_package.c

```diff
@@ -0,0 +1,32 @@
+#include <c-client/c-client.h>
+#include <stddef.h>
+
+void mm_searched(MAILSTREAM* stream, unsigned long number) {}
```

Comment:
What are these symbols for?

Reply:
Added an explaining comment in test code via https://github.com/conan-io/conan-center-index/pull/8901/commits/85ea629bcbb650d21787fac891261662eb0f8915